cubingusa.py

```diff
@@ -55,6 +55,7 @@
   webapp2.Route('/about/contact',
                 handler=ContactHandler(AppSettings.Get().contact_email, 'contact.html', 'CubingUSA'),
                 name='contact'),
+  webapp2.Route('/newengland', handler=BasicHandler('newengland.html'), name='newengland'),
```

Comment:
I'd expect this diff to introduce a new file `newengland.html`. Did that file already exist, or did you possibly forget to add it to this PR?

Reply:
You're right- I forgot to git add before committing. I just committed the actual html file.